src/firebase/AuthService.ts

```diff
@@ -0,0 +1,33 @@
+import firebase from 'firebase/app';
+import 'firebase/auth';
+
+class AuthService {
+    provider: firebase.auth.OAuthProvider;
+    accessToken: string | undefined;
+    idToken: string | undefined;
+
+    constructor() {
+        this.provider = new firebase.auth.OAuthProvider('microsoft.com');
+        this.provider.setCustomParameters({
+            // Force re-consent.
+            prompt: 'consent',
```

Comment:
We might need to set the prompt to `select_account`; this prevents problems with users having multiple accounts signed in at the same time.

Reply:
sure, did this